Client/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -356,16 +353,16 @@ class SearchViewController: SiteTableViewController, LoaderListener {
           let isSuggestClientError = error.domain == SearchSuggestClientErrorDomain
 
           switch error.code {
-          case NSURLErrorCancelled where error.domain == NSURLErrorDomain:
-            // Request was cancelled. Do nothing.
-            break
-          case SearchSuggestClientErrorInvalidEngine where isSuggestClientError:
-            // Engine does not support search suggestions. Do nothing.
-            break
-          case SearchSuggestClientErrorInvalidResponse where isSuggestClientError:
-            print("Error: Invalid search suggestion data")
-          default:
-            print("Error: \(error.description)")
+            case NSURLErrorCancelled where error.domain == NSURLErrorDomain:
+              // Request was cancelled. Do nothing.
+              break
+            case SearchSuggestClientErrorInvalidEngine where isSuggestClientError:
+              // Engine does not support search suggestions. Do nothing.
+              break
+            case SearchSuggestClientErrorInvalidResponse where isSuggestClientError:
+              print("Error: Invalid search suggestion data")
+            default:
+              print("Error: \(error.description)")
```

Comment:
extra indentations

Reply:
This is caused by auto indent after rebasing the 2-space change. I will fix all changes.

Reply:
Fix at https://github.com/brave/brave-ios/pull/5157/commits/f1acef3104359c6e8db06fa7cddd1ab1a1deebdc